maistra/run-ci.sh

```diff
@@ -1,36 +1,7 @@
 #!/bin/bash
 
-set -e
-set -o pipefail
-set -x
-
-export CC=clang CXX=clang++
-
-ARCH=$(uname -p)
-if [ "${ARCH}" = "ppc64le" ]; then
-  ARCH="ppc"
-fi
-export ARCH
-
-export BUILD_SCM_REVISION="Maistra PR #${PULL_NUMBER:-undefined}"
-export BUILD_SCM_STATUS="SHA=${PULL_PULL_SHA:-undefined}"
-
-COMMON_FLAGS="\
-    --config=clang \
```

Comment:
Note: we drop `--config=clang` in this change; bit I think that's fine, as we seem to default to it anyway.